SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -518,6 +553,21 @@ void USpatialActorChannel::SetChannelActor(AActor* InActor)
 		// Inform USpatialNetDriver of this new actor channel/entity pairing
 		NetDriver->AddActorChannel(EntityId, this);
 	}
+
+	// Set up the shadow data for the handover properties. This is used later to compare the properties and send only changed ones.
+	check(!HandoverShadowDataMap.Contains(InActor));
+
+	// Create the shadow map, and store a quick access pointer to it
+	ActorHandoverShadowData = &HandoverShadowDataMap.Add(InActor, MakeShared<TArray<uint8>>()).Get();
+	InitializeHandoverShadowData(*ActorHandoverShadowData, InActor);
+
+	// Assume that all the replicated static components are already set as such. This is checked later in ReplicateSubobject.
```

Comment:
You can remove this comment since the handover logic actually doesn't care if the components are replicated.